cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/values/SensitiveValue.java

```diff
@@ -68,7 +79,12 @@ public void encrypt() {
     protected String encrypt(Serializable originalContent) {
         byte[] serialized = serialize(originalContent);
         String serializedAsString = Base64.encodeBase64String(serialized);
-        return EncryptionProvider.get().encrypt(serializedAsString.toCharArray());
+        Encryption encryption = EncryptionProvider.get();
```

Comment:
@lucian-cm @Bonczidai based on what parameter should I make the choice to decrypt or to deobfuscate?
or you would like to duplicate code from the encryptor?